src/Avalonia.Visuals/Media/Fonts/FontAsset.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+
+namespace Avalonia.Media.Fonts
+{
+    internal class FontAsset
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FontAsset"/> class.
+        /// </summary>
+        /// <param name="source">The source.</param>
+        public FontAsset(Uri source)
```

Comment:
This class just contains a single `Uri` - why is this abstraction needed? Why not just pass a `Uri`?